docs/dev/build_raspbian.md

```diff
@@ -1,6 +1,6 @@
 # Build OpenVINO™ Runtime for Raspbian Stretch OS
 
-> **NOTE**: [ARM CPU plugin](https://github.com/openvinotoolkit/openvino_contrib/tree/master/modules/arm_plugin) are supported. The detailed instruction how to build ARM plugin is available in [OpenVINO contrib wiki](https://github.com/openvinotoolkit/openvino_contrib/wiki/How-to-build-ARM-CPU-plugin).
+> **NOTE**: Since 2023.0 release, you can compile [OpenVINO Intel CPU plugin](https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/intel_cpu) on ARM platforms.
 
```

Comment:
on this page, "Cross Compilation Using Docker" does not refer to a real page

Reply:
@ilya-lavrenov I suggest to delete the section "Cross Compilation Using Docker" at all, since we don't have any pre-configured Dockerfile for now.

Reply:
> @ilya-lavrenov I suggest to delete the section "Cross Compilation Using Docker" at all, since we don't have any pre-configured Dockerfile for now.

I have removed the section with the https://github.com/openvinotoolkit/openvino/pull/18764/commits/5bb6146348d1ae92c210e8756f7df28c4c96e804 commit.